subgraphs/spotify/src/utils/mocks.ts

```diff
@@ -55,21 +50,25 @@ export class MockedSpotifyDataSource implements SpotifyDataSource {
     id: string,
     params?: { market?: string }
   ): Promise<Spotify.Object.Album> {
-    throw new Error('Method not implemented.');
+    const album = mocks.albums.find((t) => t.id == id);
+    if (album && params?.market) {
+      if (album.available_markets.includes(params.market)) return album;
+    } else if (album) return album;
+    else return null;
   }
 
   async getAlbums(params: {
     ids: string;
     market?: string;
   }): Promise<Spotify.Object.List<'albums', Spotify.Object.Album>> {
-    throw new Error('Method not implemented.');
+    return { albums: mocks.albums };
   }
 
   async getAlbumTracks(
     id: string,
     params?: { limit?: number; offset?: number; market?: string }
   ): Promise<Spotify.Object.Paginated<Spotify.Object.TrackSimplified>> {
-    throw new Error('Method not implemented.');
+    return mocks.albums.find((t) => t.id == id)?.tracks;
```

Comment:
We should probably respect the `offset` and `limit` here:

```suggestion
    const tracks = mocks.albums.find((album) => album.id === id) ?? [];

    return tracks.slice(offset, offset + limit);
```